.vscode/settings.json

```diff
@@ -11,5 +11,6 @@
   "typescript.preferences.quoteStyle": "single",
   "prettier.singleQuote": true,
   "cSpell.words": ["adal", "uninitialize"],
-  "jest.jestCommandLine": "pnpm test -- --"
+  "jest.jestCommandLine": "pnpm test -- --",
+  "jest.autoRun": { "watch": "true", "onStartup": ["all-tests"] }
```

Comment:
What if the person pulled the repo at first time and he hasn't run pnpm install to install the jest? Would that also work?

Reply:
The tests won't and can't be run if jest isn't installed. That is also the current behavior.